test/test_mps.py

```diff
@@ -3332,8 +3332,14 @@ def helper(shape_x, shape_y, shape_z):
             self.assertEqual(cat, cat_cpu)
 
         helper([2, 2, 4, 5], [2, 3, 4, 5], [2, 5, 4, 5])
-        # Empty test - Currently failing! Empty tensor not handled!
-        # helper([0, 2, 4, 5], [2, 0, 4, 5], [2, 5, 0, 5])
+        helper([2, 2, 6, 5], [2, 3, 6, 5], [2, 5, 6, 5])
```

Comment:
Can this be tested via OpInfo below?

Reply:
Right, So this specific test without empty tensors can be tested with OpInfo. But this test-case in-general isolated few issues which were missed by TestConsistency, that's why I left it in.